.github/workflows/build.yaml

```diff
@@ -0,0 +1,28 @@
+name: Build and test operator images on push to every branch except main and pull requests
+on:
+  pull_request:
+    branches: [main]
+  push:
```

Comment:
I think you can just remove this entire push block and it will not run on any push

Reply:
But, I want it to run on all branches/forks (except main) when something is pushed

Reply:
When will you be pushing branches that aren't main?